data/Viral kinetics of SARS-CoV-2/data.yaml

```diff
@@ -0,0 +1,105 @@
+analytes:
+  throatswab_SARSCoV2:
+    biomarker: SARS-CoV-2
+    description: SARS-CoV-2 RNA genome copy concentration in throat swab samples.
+      Specimens were collected from all patients at least 2 days after hospitalization
+      and physicians checked their symptoms and signs daily.
+    gene_target: unknown
+    limit_of_detection: unknown
+    limit_of_quantification: 1000
+    reference_event: PatientID
```

Comment:
For 3 presymptomatic cases, this will be `symptom onset` and for 10 asymptomatic cases, this will be `confirmation date`. You may need to create two `analyte`s for presymptomatic and asymptomatic cases.